### PR TITLE
Add test for BI-2000. Add bad ontology sample.

### PR DESCRIPTION
**Pull request by cvacalares**
# Description
Add test for BI-2000.

# Testing
[_Please include a link to a successful run of TAF for this change_](https://github.com/Breeding-Insight/taf/actions/runs/11047171789)

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have tested my code and ensured it meets the acceptance criteria of the story
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to documentation

### CODE REVIEW

File: src/features/BI-2000.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        And user selects "Ontology" in navigation
+        And user selects "Manage Ontology" button
+        And user selects "Import file" menu item
+        And user uploads Ontology "Bad.xls" file
+		And user selects 'Import' button
+        Then user can see banner contains "Error(s) detected in file, Bad.xls . (See details below.) Import cannot proceed."
+        And user uploads Ontology "test01-ontology.xls" file
+        And user selects 'Import' button
+		When user selects "Confirm" button
+        Then user can see banner contains "Imported ontology terms have been added"
```
Comment (HMS17) [inline src/features/BI-2000.feature:17]: Given the acceptance criteria includes "THEN the user can see the error message disappear when a valid file is uploaded." should there be a "user can see banner does NOT contain 'THEN the user can see the error message disappear when a valid file is uploaded. '" clause?

**Review by HMS17 — CHANGES_REQUESTED** [review]

**Comment by cvacalares** [comment]
I have updated to add the "Then user cannot see banner contains "Error(s) detected in file, Bad.xls . (See details below.) Import cannot proceed.".

File: src/features/BI-2000.feature (hunk 1 above)
Comment (cvacalares) [reply src/features/BI-2000.feature]: I have updated to add the "Then user cannot see banner contains "Error(s) detected in file, Bad.xls . (See details below.) Import cannot proceed.".

**Review by HMS17 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "@cucumber/cucumber": "^9.4.0",
     "@cucumber/pretty-formatter": "*",
     "@slime/stopwatch": "^1.0.5",
-    "chromedriver": "^127.0.0",
+    "chromedriver": "^129.0.0",
```
[the remark this answers is not in the document]
Comment (cvacalares) [reply package.json:34]: That I'm not aware. I usually update the package.json.

[the remark this answers is not in the document]
Comment (mlm483) [reply package.json:34]: Well, it looks like it ran without any chromedriver issues.

**Review by mlm483 — APPROVED** [review]

**Pull request merged**